### ISSUE TITLE
quadratic file name size

### ISSUE THREAD

**Issue by dours** [cqfn/eo#501]
Names of some of the generated files are N^2, where N is the number of hierarchy levels, for example, this error

EOobject1Test.java:[128,14] error while writing EOorg.EOeolang.EOobject1Test.EOobject1Test$EOobject2.EOobject1Test$EOobject2$EOobject3.EOobject1Test$EOobject2$EOobject3$EOobject4.EOobject1Test$EOobject2$EOobject3$EOobject4$EOobject5.EOobject1Test$EOobject2$EOobject3$EOobject4$EOobject5$EOobject6.EOobject1Test$EOobject2$EOobject3$EOobject4$EOobject5$EOobject6$EOobject7: /home/bogus/py2eo/test/target/test-classes/EOorg/EOeolang/EOobject1Test$EOobject1Test$EOobject2$EOobject1Test$EOobject2$EOobject3$EOobject1Test$EOobject2$EOobject3$EOobject4$EOobject1Test$EOobject2$EOobject3$EOobject4$EOobject5$EOobject1Test$EOobject2$EOobject3$EOobject4$EOobject5$EOobject6$EOobject1Test$EOobject2$EOobject3$EOobject4$EOobject5$EOobject6$EOobject7.class: File name too long

happens for this source
```
[] > object1
  [] > object2
    [] > object3
      [] > object4
        [] > object5
          [] > object6
            [] > object7
              [] > object8
  TRUE > @
```

**Comment by yegor256** [comment]
@dours I can't reproduce this bug. Need more information...

**Comment by yegor256** [comment]
@dours reproduced. let me try to fix it.

**Comment by yegor256** [comment]
@dours fixed in 1bf13699

**Comment by dours** [comment]
@yegor256 just changing hash in pom.xml does not help (cannot download stdout.eo). Tests of the master pass locally, so the bug is definitely fixed.

**Comment by yegor256** [comment]
@dours I believe it's fixed, try version 0.21.3

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `DEV` is banned at [#501](https://github.com/cqfn/eo/issues/501); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))
